script_business_requirement_category/README.rst

```diff
@@ -0,0 +1,48 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
@elicoidal Do you mean for this occasion is not necessary a README.rst file? Should I remove it?